packages/daimo-expo-passkeys/src/utils.ts

```diff
@@ -0,0 +1,73 @@
+export type CreateRequest = {
+  domain: string;
+  challengeB64: string;
+  accountName: string;
+};
+
+export type CreateResult = {
+  rawClientDataJSONB64: string;
+  rawAttestationObjectB64: string;
+};
+
+export type SignRequest = {
+  domain: string;
+  challengeB64: string;
+};
+
+export type SignResult = {
+  accountName: string;
+  rawClientDataJSONB64: string;
+  rawAuthenticatorDataB64: string;
+  signatureB64: string;
+};
+
+export function toBase64URL(valueBase64: string): string {
+  return valueBase64.replace(/\+/g, "-").replace(/\//g, "_").replace(/=+$/, "");
+}
+
+export function toBase64(valueBase64URL: string): string {
+  const m = valueBase64URL.length % 4;
+  return valueBase64URL
+    .replace(/-/g, "+")
+    .replace(/_/g, "/")
+    .padEnd(valueBase64URL.length + (m === 0 ? 0 : 4 - m), "=");
```

Comment:
base64 url always has 0-2 = at the end
this makes it look like 0-3, but i think it's actually right, the case where m=1 never happens with valid input

Reply:
yeah, seems right; taken from https://stackoverflow.com/questions/5234581/base64url-decoding-via-javascript